index.html

```diff
@@ -287,6 +287,37 @@ <h2>
         and again as an example, the user agent could <a>install</a> the web
         application into a list of bookmarks within the user agent itself.
       </p>
+      <p>
+        A {{Document}} may either be <dfn>installable</dfn> or not. The initial
+        state of a document is not <a>installable</a>.
+      </p>
+      <p>
+        At any time, the user agent MAY perform the <dfn>steps to determine
+        installability of the document</dfn>:
+      </p>
+      <ol>
+        <li>Let <var>manifest</var> and <var>manifest URL</var> be the result
+        of <a>obtaining the manifest</a>.
+        </li>
+        <li>If <a>obtaining the manifest</a> results in an error, the user
+        agent MAY either:
+          <ol>
+            <li>Fall back to using the <a>top-level browsing context</a>
+            {{Document}}'s metadata to to populate <var>manifest</var> in a
+            user-agent-specific way (e.g., setting
+            |manifest|.{{WebAppManifest/name}} to the document <a data-cite=
+            "HTML#the-title-element">`title`</a>) and considering the document
+            <a>installable</a>.
+            </li>
+            <li>Or, consider the document not <a>installable</a>.
+            </li>
+          </ol>
+        </li>
+        <li>Otherwise, the {{Document}} MAY be considered <a>installable</a>
+        (at the user agent's discretion; see <a href="#installability-signals">
```

Comment:
This should auto-expand to the right section. 
```suggestion
        (at the user agent's discretion; see [[[#installability-signals]]].
```